src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,17 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return Arrays.stream(numbers.stream()
+                        .map(number -> number.split(","))
+                        .flatMap(Arrays::stream)
+                        .toArray(String[]::new))
+                .mapToInt(Integer::parseInt)
+                .boxed()
+                .filter(element -> element % 2 == 0)
+                .distinct()
+                .sorted()
+                .findFirst()
+                .orElseThrow(() -> new RuntimeException("Can't get min value from list" + numbers));
```

Comment:
```suggestion
        return numbers.stream()
                .map(number -> number.split(","))
                .flatMap(Arrays::stream)
                .mapToInt(Integer::parseInt)
                .filter(element -> element % 2 == 0)
                .sorted()
                .findFirst()
                .orElseThrow(() -> new RuntimeException("Can't get min value from list" + numbers));
```
or we can use `.min()` instead of `.sorted().findFirst()`